Update privacy.md (#709)
publiccodenet/about: organization/privacy.md

We keep contact details for:

* people who have signed up to get our news

> * people who participate in our codebase communities (e.g. Discourse and mailing lists)

Reviewer: ```suggestion
* people who participate in our codebase communities
```

I don't think this specificity adds clarity or is meaningful.

Also no abbreviations please :)

Author: It now kinda sounds like no matter what they are doing in or with our codebases, we will use what data we have of them, which is also not what we want to say with this  @ElenaFdR has a suggestion for a good middle road :)

Reviewer 2: I've made an alternate suggestion. I do think that explaining what type of participation leads to potential tracking is helpful.
```suggestion
* people who participate in our codebase communities (for example by joining mailing lists)
```
